src/main/java/com/icthh/xm/uaa/domain/User.java

```diff
@@ -1,15 +1,21 @@
 package com.icthh.xm.uaa.domain;
 
+import static com.google.common.collect.Iterables.getFirst;
+import static com.google.common.collect.Iterables.getLast;
+import static org.apache.commons.collections.CollectionUtils.isNotEmpty;
+
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.icthh.xm.uaa.domain.converter.MapToStringConverter;
-import com.icthh.xm.uaa.validator.JsonData;
+import com.icthh.xm.uaa.domain.converter.RoleKeyConverter;
 import com.icthh.xm.uaa.repository.converter.OtpChannelTypeAttributeConverter;
-import lombok.Getter;
```

Comment:
please do not reformat imports, it is unnecessary here

Reply:
removed reformatting.  But git does not see it